src/QsCompiler/LanguageServer/LanguageServer.cs

```diff
@@ -581,10 +581,13 @@ public object OnTextDocumentReferences(JToken arg)
         [JsonRpcMethod(Methods.TextDocumentCodeActionName)]
         public object OnCodeAction(JToken arg)
         {
-            Command BuildCommand(string title, WorkspaceEdit edit)
+            CodeAction CreateAction(string title, WorkspaceEdit edit)
```

Comment:
You could turn this into an expression function:

```csharp
CodeAction CreateAction(string title, WorkspaceEdit edit) =>
    new CodeAction { Title = title, Edit = edit };
```

Reply:
or move it inline into the LINQ expression, since it's pretty small.

Reply:
Sounds good. I'll leave as is for now to avoid resetting the approval (including the one from the VS SDK team), but if another change is required I'll take the suggestion as part of it.

Reply:
On the compiler repo, approvals aren't reset if new commits are pushed.